.gitignore

```diff
@@ -20,3 +20,4 @@ contrib/build/*/tmp/*
 .pkg
 .kube
 .var
+docs/certs
```

Comment:
Path looks weird, but whatever

Reply:
yea, the install directions for 1.7 in this PR are tailored toward operating the install out of this directory. can change it in a follow-up if needed